src/gsy_e/models/area/scm_dataclasses.py

```diff
@@ -261,6 +275,7 @@ class AreaFees:
     grid_export_fee_const: float = 0.0
     grid_fees_reduction: float = 0.0
     per_kWh_fees: Dict[str, FeeContainer] = field(default_factory=dict)
+    per_kW_fees: Dict[str, ContractedPowerFeeContainer] = field(default_factory=dict)
```

Comment:
I think you need to add this to the `prices_as_dict` in order to add those fees to the bill.

Reply:
Indeed. This PR is following shortly